ethdb/database.go

```diff
@@ -40,9 +40,9 @@ type Deleter interface {
 	Delete(key []byte) error
 }
 
-// DbEventLogger wraps Put and Delete to serve as a recipient
-// for batch replays
-type DbEventLogger interface {
+// Replayee wraps basic batch operations to allow replaying an existing batch
+// on top of multiple databases.
+type Replayee interface {
```

Comment:
I think it's wrong to call this `Replayee` because the operations aren't necessarily related to batch replay.
Maybe we should just remove `Deleter` and put `Delete` into the `Writer` interface. 

The `Reader` / `Writer` split makes sense because a particular piece of code may have 'write permission'
or not. Allowing writes but not deletes doesn't come up often.

Reply:
I've merged Writer and Deleter, got rid of Replayee. PTAL at the last commit.